README.md

```diff
@@ -76,39 +95,20 @@ This bundle also provides router caching, to help speed that section up. To enab
 
 ```yml
 cache:
-    router:
-        enabled: true
-        instance: default
+  router:
+    enabled: true
+    service_id: cache.provider.acme_redis_cache
 ```
 
-If you change any of your routes, you will need to clear all of the route_* keys in your cache.
-
-
-#### To Use
+If you change any of your routes, you will need to clear the cache. If you use a cache implementation that supports 
+tagging (implements [TaggablePoolTrait](https://github.com/php-cache/taggable-cache/blob/master/src/TaggablePoolInterface.php))
+you can clear the cache tagged with `routing`.
 
-To use this with doctrine's entity manager, just make sure you have `useResultCache` and/or `useQueryCache` set to true. If you want to use the user cache, just grab the service out of the container like so:
```

Comment:
If you don't specify that the user has to use these when building a query, it won't cache queries or results